curl/boring ssl link error on linux

Trying to develop on `main` on my local box I get the following link error. Any ideas?

```
$ bazel test test:eventuals
INFO: Analyzed target //test:eventuals (0 packages loaded, 0 targets configured).
INFO: Found 1 test target...
ERROR: ${PWD}/test/BUILD.bazel:54:8: Linking test/eventuals failed: (Exit 1): gcc failed: error executing command /usr/bin/gcc @bazel-out/k8-fastbuild/bin/test/eventuals-2.params

Use --sandbox_debug to see verbose messages from the sandbox
bazel-out/k8-fastbuild/bin/external/com_github_curl_curl/libcurl_boringssl_linux_macos/lib/libcurl.a(ldap.c.o):ldap.c:function ldap_do: error: undefined reference to 'ldap_url_parse'
bazel-out/k8-fastbuild/bin/external/com_github_curl_curl/libcurl_boringssl_linux_macos/lib/libcurl.a(ldap.c.o):ldap.c:function ldap_do: error: undefined reference to 'ldap_set_option'
bazel-out/k8-fastbuild/bin/external/com_github_curl_curl/libcurl_boringssl_linux_macos/lib/libcurl.a(ldap.c.o):ldap.c:function ldap_do: error: undefined reference to 'ldap_set_option'
bazel-out/k8-fastbuild/bin/external/com_github_curl_curl/libcurl_boringssl_linux_macos/lib/libcurl.a(ldap.c.o):ldap.c:function ldap_do: error: undefined reference to 'ldap_init'
bazel-out/k8-fastbuild/bin/external/com_github_curl_curl/libcurl_boringssl_linux_macos/lib/libcurl.a(ldap.c.o):ldap.c:function ldap_do: error: undefined reference to 'ldap_simple_bind_s'
bazel-out/k8-fastbuild/bin/external/com_github_curl_curl/libcurl_boringssl_linux_macos/lib/libcurl.a(ldap.c.o):ldap.c:function ldap_do: error: undefined reference to 'ldap_set_option'
bazel-out/k8-fastbuild/bin/external/com_github_curl_curl/libcurl_boringssl_linux_macos/lib/libcurl.a(ldap.c.o):ldap.c:function ldap_do: error: undefined reference to 'ldap_simple_bind_s'
bazel-out/k8-fastbuild/bin/external/com_github_curl_curl/libcurl_boringssl_linux_macos/lib/libcurl.a(ldap.c.o):ldap.c:function ldap_do: error: undefined reference to 'ldap_search_s'
bazel-out/k8-fastbuild/bin/external/com_github_curl_curl/libcurl_boringssl_linux_macos/lib/libcurl.a(ldap.c.o):ldap.c:function ldap_do: error: undefined reference to 'ldap_first_entry'
bazel-out/k8-fastbuild/bin/external/com_github_curl_curl/libcurl_boringssl_linux_macos/lib/libcurl.a(ldap.c.o):ldap.c:function ldap_do: error: undefined reference to 'ldap_get_dn'
bazel-out/k8-fastbuild/bin/external/com_github_curl_curl/libcurl_boringssl_linux_macos/lib/libcurl.a(ldap.c.o):ldap.c:function ldap_do: error: undefined reference to 'ldap_memfree'
bazel-out/k8-fastbuild/bin/external/com_github_curl_curl/libcurl_boringssl_linux_macos/lib/libcurl.a(ldap.c.o):ldap.c:function ldap_do: error: undefined reference to 'ldap_first_attribute'
bazel-out/k8-fastbuild/bin/external/com_github_curl_curl/libcurl_boringssl_linux_macos/lib/libcurl.a(ldap.c.o):ldap.c:function ldap_do: error: undefined reference to 'ldap_get_values_len'
bazel-out/k8-fastbuild/bin/external/com_github_curl_curl/libcurl_boringssl_linux_macos/lib/libcurl.a(ldap.c.o):ldap.c:function ldap_do: error: undefined reference to 'ldap_set_option'
bazel-out/k8-fastbuild/bin/external/com_github_curl_curl/libcurl_boringssl_linux_macos/lib/libcurl.a(ldap.c.o):ldap.c:function ldap_do: error: undefined reference to 'ldap_simple_bind_s'
bazel-out/k8-fastbuild/bin/external/com_github_curl_curl/libcurl_boringssl_linux_macos/lib/libcurl.a(ldap.c.o):ldap.c:function ldap_do: error: undefined reference to 'ldap_err2string'
bazel-out/k8-fastbuild/bin/external/com_github_curl_curl/libcurl_boringssl_linux_macos/lib/libcurl.a(ldap.c.o):ldap.c:function ldap_do: error: undefined reference to 'ldap_value_free_len'
bazel-out/k8-fastbuild/bin/external/com_github_curl_curl/libcurl_boringssl_linux_macos/lib/libcurl.a(ldap.c.o):ldap.c:function ldap_do: error: undefined reference to 'ldap_memfree'
bazel-out/k8-fastbuild/bin/external/com_github_curl_curl/libcurl_boringssl_linux_macos/lib/libcurl.a(ldap.c.o):ldap.c:function ldap_do: error: undefined reference to 'ber_free'
bazel-out/k8-fastbuild/bin/external/com_github_curl_curl/libcurl_boringssl_linux_macos/lib/libcurl.a(ldap.c.o):ldap.c:function ldap_do: error: undefined reference to 'ldap_msgfree'
bazel-out/k8-fastbuild/bin/external/com_github_curl_curl/libcurl_boringssl_linux_macos/lib/libcurl.a(ldap.c.o):ldap.c:function ldap_do: error: undefined reference to 'ldap_free_urldesc'
bazel-out/k8-fastbuild/bin/external/com_github_curl_curl/libcurl_boringssl_linux_macos/lib/libcurl.a(ldap.c.o):ldap.c:function ldap_do: error: undefined reference to 'ldap_unbind_s'
bazel-out/k8-fastbuild/bin/external/com_github_curl_curl/libcurl_boringssl_linux_macos/lib/libcurl.a(ldap.c.o):ldap.c:function ldap_do: error: undefined reference to 'ldap_value_free_len'
bazel-out/k8-fastbuild/bin/external/com_github_curl_curl/libcurl_boringssl_linux_macos/lib/libcurl.a(ldap.c.o):ldap.c:function ldap_do: error: undefined reference to 'ldap_memfree'
bazel-out/k8-fastbuild/bin/external/com_github_curl_curl/libcurl_boringssl_linux_macos/lib/libcurl.a(ldap.c.o):ldap.c:function ldap_do: error: undefined reference to 'ldap_next_attribute'
bazel-out/k8-fastbuild/bin/external/com_github_curl_curl/libcurl_boringssl_linux_macos/lib/libcurl.a(ldap.c.o):ldap.c:function ldap_do: error: undefined reference to 'ber_free'
bazel-out/k8-fastbuild/bin/external/com_github_curl_curl/libcurl_boringssl_linux_macos/lib/libcurl.a(ldap.c.o):ldap.c:function ldap_do: error: undefined reference to 'ldap_next_entry'
bazel-out/k8-fastbuild/bin/external/com_github_curl_curl/libcurl_boringssl_linux_macos/lib/libcurl.a(ldap.c.o):ldap.c:function ldap_do: error: undefined reference to 'ldap_err2string'
bazel-out/k8-fastbuild/bin/external/com_github_curl_curl/libcurl_boringssl_linux_macos/lib/libcurl.a(ldap.c.o):ldap.c:function ldap_do: error: undefined reference to 'ldap_err2string'
bazel-out/k8-fastbuild/bin/external/com_github_curl_curl/libcurl_boringssl_linux_macos/lib/libcurl.a(ldap.c.o):ldap.c:function ldap_do: error: undefined reference to 'ldap_memfree'
collect2: error: ld returned 1 exit status
Target //test:eventuals failed to build
```

--- a/3rdparty/bazel-rules-curl/repos.bzl
+++ b/3rdparty/bazel-rules-curl/repos.bzl
@@ -28,7 +28,7 @@ def repos(external = True, repo_mapping = {}):
             git_repository,
             name = "com_github_3rdparty_bazel_rules_curl",
             remote = "https://github.com/3rdparty/bazel-rules-curl",
-            commit = "3c46e684ee629bf2502946ccdd369b1f9d42de08",
-            shallow_since = "1642609159 +0300",
+            commit = "48cf05764bea5fa4a9e539efd837608bbf3a69de",
+            shallow_since = "1651695807 +0200",
             repo_mapping = repo_mapping,
         )

